### ISSUE TITLE
Font for source code isn't very attractive

### ISSUE THREAD

**Issue by cormullion** [pat-alt/quarto-juliacon-proceedings#39]
I know it's an aesthetic thing, but I don't think the monospace font is very attractive:

<img width="804" alt="Screenshot 2024-01-07 at 16 25 35" src="https://github.com/pat-alt/quarto-juliacon-proceedings/assets/52289/4674cb86-014f-433b-9b6a-97dc167fcde6">

Latin Modern Mono?

It _almost_ looks like it's  gone wrong (double-spaced or something).

Feel free to ignore! :)

**Comment by pat-alt** [comment]
So here I'm actually relying on the style file that's currently used for JuliaCon Proceedings. For now I've been trying to replicate the existing LaTeX template as closely as possible to make this extension viable for publications. 

In the past, I've played around with another extension that uses JuliaMonofont and Julia colors for code blocks in HTML documents: https://www.paltmeyer.com/quarto-julia/examples/article/#code. Could draw inspiration from that for a HTML version of JuliaCon Proceedings. That would be easy enough to add here, which is one of the things that make Quarto so attractive imo 😃

**Comment by cormullion** [comment]
Yes, I did notice it in the old proceedings, but if there's an opportunity to  improve on them... 

I really like your Julia-colored take on the design...

Most monospaced fonts would look better than Latin Modern. (I confess I'm a little [biased against ComputerModern](https://cormullion.github.io/pages/2023-06-22-modern/) anyway... 😀 )

**Comment by pat-alt** [comment]
I agree! 

> Yes, I did notice it in the old proceedings, but if there's an opportunity to improve on them...

To drive that point home, I've created a separate website that presents the PDF and HTML versions of the paper side-by-side: https://www.paltmeyer.com/quarto-julia-website/examples/juliacon/counterfactual-explanations/.

Planning to submit this to JuliaCon 2024 as well, so if you should have any more suggestions, please don't hold back 😄
